dojo/tools/trivy/parser.py

```diff
@@ -55,17 +55,20 @@ def get_description_for_scan_types(self, scan_type):
         return "Import trivy JSON scan report."
 
     def convert_cvss_score(self, raw_value):
-        val = float(raw_value)
-        if val == 0.0:
+        if raw_value is None:
```

Comment:
Should we adapt the raw_value== None to score "Low"?

Reply:
We have "Info" which feels like a better fit than "Low" for a raw_value == None

Reply:
in my opinion None should not equal Low as it misrepresents the risk associated with the finding.  If the tool has not assigned a value, then we should likely interpret that as Informational.

Reply:
Ok, then I will leave it like it is. As soon as a severity is available, the old one will be closed anyway as severity is also in the deduplication settings in settings.dist.py